code/src/App.js

```diff
@@ -1,9 +1,68 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+
+import { MessageList } from "./components/MessageList";
+import { MessageInput } from "./components/MessageInput";
+import { Header } from "./components/Header";
+import { Footer } from "./components/Footer";
+import { Loader } from "./components/Loader";
+import { MESSAGE_URL } from "./urls";
+
+import "./app.css";
 
 export const App = () => {
+  const [messages, setMessages] = useState([]);
+  const [loader, setLoader] = useState(true);
+
+  useEffect(() => {
+    fetchMessages();
+  }, []);
+
+  const fetchMessages = () => {
+    fetch(MESSAGE_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        setTimeout(() => {
+          setLoader(false);
+          setMessages(data);
+        }, 3000);
```

Comment:
Not sure why you need 3000 ms to execute your code - it's not necessary at all

Reply:
It's just because I was so happy with the rainbow animation! :D